dashboard/app/assets/javascripts/angular/testExecutor.service.js

```diff
@@ -25,7 +25,7 @@
           method: 'DELETE'
         }
       });
-      var testExecUpdateIntervalPeriod = 5; // seconds
+      var testExecUpdateIntervalPeriod = 1; // seconds
```

Comment:
The amount of work the dashboard server is doing per request is constant (it is simply returning the state of the test execution as obtained from the test server. ) The number of requests will of course change based on the number of test executions happening. Since there is no computation involved to serve each request, 1 second seems fine?